src/test/resources/MappingHelper/Custom_ListedElectronicSerial.json

```diff
@@ -1,313 +1,316 @@
-            "translateDefault": true
-          }
-        },
-        {
-          "field": "VENDOR",
-          "dataSource": {
-            "default": "GOBI",
-            "translation": "lookupOrganization",
-            "translateDefault": true
-          }
-        },
-        {
-          "field": "MATERIAL_SUPPLIER",
-          "dataSource": {
-            "default": "GOBI",
-            "translation": "lookupOrganization",
-            "translateDefault": true
-          }
-        },
-        {
-          "field": "VENDOR_REF_NO",
-          "dataSource": {
-            "from": "//YBPOrderKey"
-          }
-        },
-        {
-          "field": "VENDOR_REF_NO_TYPE",
-          "dataSource": {
-            "default": "Vendor order reference number"
-          }
-        },
-        {
-          "field": "WORKFLOW_STATUS",
-          "dataSource": {
-            "default": "Open"
-          }
-        },
-        {
-          "field": "ASSIGNED_TO",
-          "dataSource": {
-            "default": "5980020f-6c3a-41e1-a16c-f449f0d8af71"
-          }
-        },
-        {
-          "field": "CANCELLATION_RESTRICTION_NOTE",
-          "dataSource": {
-            "default": "Test Data"
-          }
-        },
-        {
-          "field": "DONOR",
-          "dataSource": {
-            "default": "Test Data"
-          }
-        },
-        {
-          "field": "SELECTOR",
-          "dataSource": {
-            "default": "Test Data"
-          }
-        },
-        {
-          "field": "VOLUMES",
-          "dataSource": {
-            "default": "Test Data"
-          }
-        },
-        {
-          "field": "TAGS",
-          "dataSource": {
-            "default": "Test Data"
-          }
-        },
-        {
-          "field": "PO_LINE_DESCRIPTION",
-          "dataSource": {
-            "default": "Description from Custom Mapping"
-          }
-        },
-         {
-          "field": "LOCATION",
-          "dataSource": {
-            "from" : "//FundCode",
-            "default" : "*",
-            "translation": "lookupLocationId",
-            "translateDefault": true
-          }
-        },
-        {
-          "field": "CONTRIBUTOR_NAME_TYPE",
-          "dataSource": {
-            "default": "Test Data",
-            "translation": "lookupContributorNameTypeId",
-            "translateDefault": true
-          }
-        },
-        {
-          "field": "MATERIAL_TYPE",
-          "dataSource": {
-            "from" : "//FundCode",
-            "default" : "unspecified",
-            "translation": "lookupMaterialTypeId",
-            "translateDefault": true
-          }
-        }
-      ]
+  "orderType": "ListedElectronicSerial",
+  "mappings": [
+    {
+      "field": "ACCESS_PROVIDER",
+      "dataSource": {
+        "from": "//PurchaseOption/VendorPOCode",
+        "translation": "lookupOrganization"
+      }
+    },
+    {
+      "field": "ACQUISITION_UNIT",
```

Comment:
We already have karate test for gobi, please run it to make sure that everething is okay, also add additaional assert if needed to that karate test that checks that acq units correctly assigned.
If you have issue with local run - let me know, we will try to fix